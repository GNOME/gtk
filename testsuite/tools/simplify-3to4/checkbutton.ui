<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow" id="window1">
    <child>
      <object class="GtkBox" id="box1">
        <property name="orientation">GTK_ORIENTATION_VERTICAL</property>
        <property name="visible">1</property>
        <child>
          <object class="GtkCheckButton" id="checkbutton1">
            <property name="draw_indicator">1</property>
            <property name="hexpand">1</property>
            <property name="label">Check</property>
            <property name="visible">1</property>
          </object>
        </child>
      </object>
    </child>
  </object>
</interface>
