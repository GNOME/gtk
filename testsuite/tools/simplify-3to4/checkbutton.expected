<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkWindow" id="window1">
    <property name="child">
      <object class="GtkBox" id="box1">
        <property name="orientation">vertical</property>
        <child>
          <object class="GtkCheckButton" id="checkbutton1">
            <property name="hexpand">1</property>
            <property name="label">Check</property>
          </object>
        </child>
      </object>
    </property>
  </object>
</interface>
